### ISSUE TITLE
Selected state of Language Switch has colour contrast issues

### ISSUE THREAD

**Issue by nickytonline** [open-sauced/app#2369]
> Maybe a different issue, but are we worries about the accessibility of this white on orange color palette:
> 
> 
> 
> ![Screenshot 2023-12-18 at 8 23 23 AM](https://github.com/open-sauced/app/assets/23109390/a8ba06f4-476d-4e6b-8440-2b4200e70073)
> 
> 
> 
> Ran this through firefox's accessibility tool real quick and it pops up as inaccessible for text standards

Yeah, that's a separate issue in regards to colour contrast. This issue was specific to not being able to tab or have focus states.

_Originally posted by @nickytonline in https://github.com/open-sauced/app/issues/2365#issuecomment-1860890420_

`bg-light-orange` with the default global text colour works.

![CleanShot 2023-12-18 at 12 40 50](https://github.com/open-sauced/app/assets/833231/8f8b1348-8435-47b0-8402-0b1fd6f992bd)

![CleanShot 2023-12-18 at 12 40 50](https://github.com/open-sauced/app/assets/833231/a4ea50db-ce7e-404c-9f26-d6aeae45e5d9)

**Comment by nickytonline** [comment]
@isabensusan, to sort this out, the orange has to be really dark and it's still failing one of the contrast checks.

![CleanShot 2023-12-18 at 12 34 43](https://github.com/open-sauced/app/assets/833231/73aeac72-e1fa-4a72-8f10-ff346a5070f5)

![CleanShot 2023-12-18 at 12 34 43](https://github.com/open-sauced/app/assets/833231/7c6e17c6-5f34-4e79-9471-4338814d58f2)

Just wondering if you have a suggestion for the selected style.
